src/routes/api/cryptor-migrations/migration-router.ts

```diff
@@ -0,0 +1,115 @@
+import { Request, Response } from "express";
+import { WhereOptions } from "sequelize";
+import { Installation } from "models/installation";
+
+const MAX_BATCH_SIZE = 100;
+const MAX_TOTAL_LIMIT = 10_000;
+const SAFE_GUARD_LOOP_COUNT = 100;
+
+export const CryptorMigrationInstallationPost = async (req: Request, res: Response): Promise<void> => {
+
+	const startTime = Date.now();
+
+	let batchSize = parseInt(req.query.batchSize as string);
+	let totalLimit = parseInt(req.query.totalLimit as string);
+	const jiraHost = req.query.jiraHost as string;
+
```

Comment:
It's a post, you should be using the body, not the querystring.

You also need to validate the input and give sensible defaults if nothing is specified or error out the API call.  You also need to add limits to make sure you don't do too many.

Reply:
Agree on the post/body part, will fix it. 

`You also need to validate the input` is done bellow.